### PR TITLE
feat(build): upgrade gradle and plugins

### PR DESCRIPTION
**Pull request by Khartris**
This PR performs maintenance on the plugin's build.
Gradle is updated to v7.3.1.
Plugins are updated to their latest versions.
JCenter is replaced with corresponding repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,9 @@ plugins {
     `kotlin-dsl`
 }
 repositories {
-    jcenter()
+    maven {
+        url = uri("https://plugins.gradle.org/m2/")
+    }
```
Comment (Ingwersaft) [inline README.md]:
```suggestion
    gradlePluginPortal()
```

File: integration-token/buildSrc/build.gradle.kts [hunk 2]
```diff
@@ -2,8 +2,11 @@ plugins {
     `kotlin-dsl`
 }
 repositories {
-    jcenter()
     mavenLocal()
+    mavenCentral()
+    maven {
+        url = uri("https://plugins.gradle.org/m2/")
+    }
```
Comment (Ingwersaft) [inline integration-token/buildSrc/build.gradle.kts]:
```suggestion
    gradlePluginPortal()
```

**Review by Ingwersaft — CHANGES_REQUESTED** [review]
LGTM beside those 2 minor points 👍

File: README.md (hunk 1 above)
Comment (Khartris) [reply README.md]: done

File: integration-token/buildSrc/build.gradle.kts (hunk 2 above)
Comment (Khartris) [reply integration-token/buildSrc/build.gradle.kts]: done

**Review by Ingwersaft — APPROVED** [review]
LGTM!

**Pull request merged**